lib/pr_checker.js

```diff
@@ -460,14 +429,14 @@ class PRChecker {
   }
 
   async checkNodejsCI() {
-    let status = this.checkActionsCI();
+    let status = this.checkGitHubCI();
     if (
       this.pr.labels.nodes.some((l) => l.name === 'needs-ci') ||
       this.requiresJenkinsRun()
     ) {
       status &= await this.checkJenkinsCI();
     } else {
-      this.cli.info('Green GitHub Actions CI is sufficient');
+      this.cli.info('Green GitHub CI is sufficient');
```

Comment:
The product is called GitHub Actions, we call it GitHub Actions CI elsewhere: https://github.com/nodejs/node/blob/34fdea1c57869f097b1fc071da959280dbe44e86/doc/guides/collaborator-guide.md#L208

Reply:
There were many more occurrences of Github CI in this project. Do you want me to change all of them?